dotcom-rendering/package.json

```diff
@@ -127,6 +128,7 @@
     "compose-function": "^3.0.3",
     "compression": "^1.7.3",
     "cpy": "^8.1.2",
+    "crypto-js": "^4.1.1",
```

Comment:
This package is pretty big and doesn't tree shake according to:
https://bundlephobia.com/package/crypto-js@4.1.1

Or are we less concerned as AMP will SSR and cache - which is fair!

There's also a built-in way to do SHA256 but it doesn't support IE11 - not sure if that is an issue.
https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest#basic_example

Reply:
This is all server-side rendered anyways. No JS is passed to the client in AMP.

Reply:
Actuall, [according to the Node 14 LTS docs](https://nodejs.org/docs/latest-v14.x/api/crypto.html#crypto_hash_digest_encoding), `crypto` should cover this use case.

Reply:
Removed in #10027